### ISSUE TITLE
pip3 install numpy does not work on arm64

### ISSUE THREAD

**Issue by srohit0** [numpy/numpy#15177]
## Error Message
```
travis_time:start:0b4b9b7c
[0K$ pip3 install numpy
Collecting numpy
  Downloading https://files.pythonhosted.org/packages/31/0a/5df350c29a06835d534a6c4f5681075304da38d85f1c69e5226a635a67ce/numpy-1.18.0.zip (5.4MB)
Building wheels for collected packages: numpy
  Running setup.py bdist_wheel for numpy: started
  Running setup.py bdist_wheel for numpy: finished with status 'error'
  Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-u485u_8l/numpy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpijsuuig2pip-wheel- --python-tag cp36:
  Running from numpy source directory.
  Cythonizing sources
  Processing numpy/random/_bounded_integers.pxd.in
  Processing numpy/random/_philox.pyx
  Traceback (most recent call last):
    File "/tmp/pip-build-u485u_8l/numpy/tools/cythonize.py", line 61, in process_pyx
      from Cython.Compiler.Version import version as cython_version
  ModuleNotFoundError: No module named 'Cython'
  
  During handling of the above exception, another exception occurred:
```

**Comment by rgommers** [comment]
Hmm, that this works on ARM v7 but not ARM64 is weird, there may be something else different. Is your `pip` version up to date? You should have a recent pip version for this, probably >= 19.0

**Comment by mattip** [comment]
There are third-party wheels for ARMv7 through [piwheels](https://www.piwheels.org/), and the typical raspian install of a raspberrypi will automatically use that in addition to PyPI. As of yet I am not aware of ARM64 wheels for NumPy. There is an ongoing effort to support them via manylinux2014, but issues in our wheel building infrastructure are holding us back right now. xref gh-14886.

Currently, in order to `pip install` numpy on ARM64, you must build from source. You can `apt install` (or whatever the counterpart is for your platform).

**Comment by mattip** [comment]
xref MacPython/numpy-wheels#74

**Comment by Turbrex** [comment]
Consider uninstalling pip and using aptitude to reinstall it. Basic usage: **sudo aptitude install package**

**Comment by paroque28** [comment]
I ran into the same issue, is there any workaround yet?

**Comment by rgommers** [comment]
Thanks for updating with the fix @paroque28. This seems solved, and we have ARM64 CI plus conda binaries in Miniforge, so it's clear this works - closing.

**Comment by fkromer** [comment]
Sorry for posting into this closed issue. From which numpy version onwards are wheels for AMR64 now?

**Comment by mattip** [comment]
1.19.0 and up have aarch64 wheels for linux-on-arm. We do not have wheels for macos-on-arm yet.

**Comment by psifertex** [comment]
Is there a separate issue tracking wheels for macos?

**Comment by mattip** [comment]
Wheels for macos arm64 have been available since 1.21.0 for python3.8+. We will not be providing macos arm64 wheels for python3.7.

**Comment by psifertex** [comment]
Thanks for confirming that -- fwiw, my issue was that I needed to first `sudo python3 -mpip install --upgrade pip` to upgrade the MacOS shipping pip before it worked.
